client/components/SongBar.jsx

```diff
@@ -12,24 +12,22 @@ const SongBar = ({
   const totalMinSec = helpers.convertToMinSec(playState.totalTime);
   const nBars = songProfile.profile.length;
 
-  songProfile.profile.forEach((bar, i) => {
+  songProfile.profile.forEach((barHeight, i) => {
     locations.forEach((location, j) => {
       bars[j].push((<SongBarEntry
         position={location}
-        bar={bar}
-        number={i}
+        barHeight={barHeight}
         playState={playState}
         barFraction={i / nBars}
         handleScan={(location === 'upper') ? handleScan : () => {}}
-        handleExit={(location === 'upper') ? handleExit : () => {}}
         handleClick={(location === 'upper') ? handleClick : () => {}}
         key={`bar_${location}_${i}`}
       />
       ));
     });
   });
   const handleBarClick = (e) => {
-    console.log('in bar', Object.keys(e.target.className, e.target.id), e.screenX, e.target.offsetLeft, e.target.offsetWidth);
+    console.log('in bar', e.target.offsetLeft, e.target.offsetWidth);
```

Comment:
just a side note to remove console log when we finish our coding

Reply:
YES! This feature is still a little buggy, so I have them in there for now!